soap_api/utils.py

```diff
@@ -14,8 +14,31 @@
 log = logging.getLogger(__name__)
 import csv
 
-csv_see_dummy = f'data/bnetza_mastr_{DATA_VERSION}'
-csv_see = f'data/bnetza_mastr_{DATA_VERSION}_power-unit.csv'
+""" dummy stump for other file names """
+fname_dummy = f'data/bnetza_mastr_{DATA_VERSION}'
```

Comment:
I would prefer `fname_template` for variable name here

Reply:
ok! will change